### PR TITLE
Added Brownies

### PR DESCRIPTION
**Pull request by Nyanarchy**

### CODE REVIEW

**Comment by Knochenmark** [comment]
@Nyanarchy Hi, I saw you added 3 recipes, i just checked this one for now, because the build failed. Had to check the whole file to notice, that you added the new recipe at the wrong location. The bottom part is the `emptyRecipe` which is used when a recipe is created and for testing. You added your recipe there. You should just move your recipe up to the recipes array at line 244. Then the build should pass too. I suppose you made the same mistake for your other 2 PR's too, if you fix that I will merge them :)

**Comment by Nyanarchy** [comment]
Hey, thanks a lot for mentioning my mistake. I'm a total newbie and I was
really confused when I saw an error. Didn't know what to do about it
though. I'll try to do better, thanks!

File: src/_config/exampleRecipeList.ts [hunk 1]
```diff
@@ -241,6 +241,36 @@ export const exampleRecipeList: IRecipe[] = [
     isBookmarked: false,
     name: 'Dakos',
     preparationTime: 5
+  }
```
Comment (Knochenmark) [inline src/_config/exampleRecipeList.ts]: I think you are missing a comma here.

File: src/_config/exampleRecipeList.ts [hunk 2]
```diff
@@ -253,4 +283,5 @@ export const emptyRecipe: IRecipe = {
   isBookmarked: false,
   name: '',
   preparationTime: 0,
-}
\ No newline at end of file
+},
```
Comment (Knochenmark) [inline src/_config/exampleRecipeList.ts]: This comma isnt required, you could omit it.

**Comment by Knochenmark** [comment]
@Nyanarchy Maybe I should move that emptyRecipe into a separate file to reduce confusion.

File: src/_config/exampleRecipeList.ts [hunk 3]
```diff
@@ -241,6 +241,36 @@ export const exampleRecipeList: IRecipe[] = [
     isBookmarked: false,
     name: 'Dakos',
     preparationTime: 5
+  }
+	{
+    cookingTime: 15,
+    difficulty: Difficulty.Middle,
+    imageUrl: 'https://i2.wp.com/cafedelites.com/wp-content/uploads/2018/02/Best-Brownies-Ever-IMAGE-89.jpg?resize=980%2C1470&ssl=1',
+    ingredients: [
+'1 cup 8oz/240g butter, melted and cooled'
```
Comment (Knochenmark) [inline src/_config/exampleRecipeList.ts]: you are missing commas behind each array entry too

File: src/_config/exampleRecipeList.ts [hunk 4]
```diff
@@ -241,6 +241,36 @@ export const exampleRecipeList: IRecipe[] = [
...
+Lightly grease an 8x12-inch square baking pan* with cooking oil spray. Line with parchment paper (or baking paper); set aside.
+Combine melted butter, oil and sugars together in a medium-sized bowl. Whisk well to combine. Add the eggs and vanilla; beat until lighter in colour (another minute).
+
+Sift in flour, cocoa powder and salt. Gently fold the dry ingredients into the wet ingredients until JUST combined (do NOT over beat as doing so well affect the texture of your brownies).
+Fold in 3/4 of the chocolate pieces.
+Pour batter into prepared pan, smoothing the top out evenly, and top with remaining chocolate pieces.
+Bake for 25-30 minutes or until the centre of the brownies in the pan no longer jiggles and is just set to the touch (the brownies will keep baking in the hot pan out of the oven). If testing with a toothpick, the toothpick should come out dirty for fudge-textured brownies.
+After 10 minutes, carefully remove them out of the pan and allow to cool to room temperature before slicing into 16 brownies.`,
+    isBookmarked: false,
+    name: 'Glazed beets',
```
Comment (Knochenmark) [inline src/_config/exampleRecipeList.ts]: I guess you copied the recipe and forgot to change the name to `Brownies`

**Pull request closed without merging**